packages/@lwc/template-compiler/src/__tests__/fixtures/regression/nested-if-loops/expected.js

```diff
@@ -1,24 +1,19 @@
-import { registerTemplate } from "lwc";
+import { parseFragment, registerTemplate } from "lwc";
+const $fragment1 = parseFragment`<p${3}>Inner</p>`;
 const stc0 = [];
 function tmpl($api, $cmp, $slotset, $ctx) {
   const {
     t: api_text,
     k: api_key,
-    h: api_element,
+    st: api_static_fragment,
     i: api_iterator,
     f: api_flatten,
   } = $api;
   return $cmp.isTrue
     ? api_flatten([
         api_text("Outer"),
         api_iterator($cmp.items, function (item) {
-          return api_element(
-            "p",
-            {
-              key: api_key(0, item.id),
-            },
-            [api_text("Inner")]
-          );
+          return api_static_fragment($fragment1, api_key(1, item.id));
```

Comment:
It's interesting that the key changed from 0 > 1, I think it's because the databag being called before static content as described in [#4077](https://github.com/salesforce/lwc/issues/4077)

I don't think it'll be an observable difference tho!